esmvalcore/preprocessor/_area.py

```diff
@@ -177,7 +175,7 @@ def tile_grid_areas(cube, fx_files):
     grid_areas = None
     if fx_files:
         for key, fx_file in fx_files.items():
-            if fx_file is None:
+            if not fx_file:
                 continue
```

Comment:
@jvegasbsc can you pls briefly explain the logic? where would one get a `''` for a file variable?

Reply:
After the resolve, fx_files contains `{areacello: ""}` when a specific dataset does not have fx files. Ideally, we may want to changes the resolve so it returns None, but this will suffice in the meantime

Reply:
right! good catch!